utility/check_baseline.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+# This script sets the attenuation values based on a csv file
+
+import numpy as np
+import optparse
+import katpoint
+
+
+def remove_ants(ants,antlist,verbose=False):
+    newlist = []
+    for ant in ants:
+        if not ant.name in antlist:
+            newlist.append(ant)
+        else:
+            verbosef("%s removed from the list of antennas"%(ant.name),verbose)
+    return newlist
+
+def get_ant_array():
+    ants=['' for x in range(64)]
+    ants[0]=katpoint.Antenna('m000, -30:42:39.8, 21:26:38.0, 1086.6, 13.5, -8.264   -207.290 8.597')
```

Comment:
The are a lot of copies of our antenna coordinates a being spread over a different repos, which isn't great for maintenance.  I guess if we just need rough calculations, then it will always be good enough.  The antennas are going anywhere.   With MKAT+, there will be more to add.

Other locations include [katconfig](https://github.com/ska-sa/katconfig/tree/karoo/user/delay-models/mkat), [astrokat](https://github.com/ska-sa/astrokat/blob/master/config/mkat_antennas.yml) and I think SDP have a copy somewhere too.

Reply:
All the copies tend to be in different formats. But yes this a maintenance problem.